.github/workflows/ci.yml

```
@@ -68,9 +68,11 @@ jobs:
       matrix:
         include:
           - name: Node 18
-            NODE_VERSION: 18.19.0
+            NODE_VERSION: 18.20.3
           - name: Node 20
-            NODE_VERSION: 20.10.0
+            NODE_VERSION: 20.15.0
+          - name: Node 22
+            NODE_VERSION: 22.4.0
```

Should we remove the specific node versions and only specify the major? Like in the push adapter. We always specify exact versions for reproducibility. But I think that makes more sense for dependencies than for the node versions in the CI. For debugging, the exact Node version used can be looked up in the CI logs anyway...

```suggestion
            NODE_VERSION: 22
```

---

I agree